test/python/WMCore_t/WMBS_t/JobSplitting_t/WMBSMergeBySize_t.py

```diff
@@ -30,7 +30,8 @@ def setUp(self):
         """
         self.testInit = TestInit(__file__)
         self.testInit.setLogging()
-        self.testInit.setDatabaseConnection()
+        self.testDB = 'unittest_%s' % self.__class__.__name__
+        self.testInit.prepareDatabase(self.testDB)
```

Comment:
Eric, in order to do that every single unittests classes need to add __init__
method and properly initialize it from unittest.TestCase class.
Moving self.testDB into __init__ would be proper thing to do too.
But in order to do that we need to agree what WMCore team wants to use
for database, use single database in all tests or use individual database
within unit tests.

On  0, Eric Vaandering <notifications@github.com> wrote:
> > @@ -30,7 +30,8 @@ def setUp(self):
> >          """
> >          self.testInit = TestInit(__file__)
> >          self.testInit.setLogging()
> > -        self.testInit.setDatabaseConnection()
> > +        self.testDB = 'unittest_%s' % self.__class__.__name__
> > +        self.testInit.prepareDatabase(self.testDB)
> 
> A more limited comment. Why not move these two new lines into the setup() of the base class and use super() where needed? Same question with stuff that is repeatedly used in tearDown() 
> 
> I just satisfied my own curiously that `self.__class__.__name__` is the name of the derived class, even when accessed in the base class. Also, to be "nice" self.testDB should be defined in `__init__`, not here.
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/dmwm/WMCore/pull/6260/files#r39916189